available-connectors.json

```diff
@@ -191,6 +201,28 @@
     "supportedByStrongLoop": true
   },
   {
+    "name": "kv-redis",
+    "description": "Redis key-value connector",
+    "baseModel": "KeyValueModel",
+    "features": {
+      "discovery": false,
+      "migration": false
+    },
+    "settings": {
+      "host": {
+        "type": "string"
+      },
+      "port": {
+        "type": "number"
+      }
```

Comment:
The Redis connector has more option than just host and port, see https://github.com/luin/ioredis#connect-to-redis and https://github.com/luin/ioredis/blob/master/API.md#new_Redis_new

 - `password` of type `string`
 - `db` of type `number`

We should also support `url`, but that's not implemented yet. I opened https://github.com/strongloop/loopback-connector-kv-redis/issues/15 for that.